common/scripted_triggers/wc_scheme_triggers.txt

```diff
@@ -0,0 +1,24 @@
+﻿valid_hostile_scheme_target_trigger = {
+	is_landed = yes
```

Comment:
So I can create a scheme targeting a landless character so I assume it doesn't work.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/118825186-8722aa80-b8cb-11eb-9901-d8734caa9df3.png)

</details>